zerver/openapi/zulip.yaml

```diff
@@ -19571,6 +19612,33 @@ components:
             before Zulip 5.0 (feature level 76).
 
             A typical failed json response for when user's organization is deactivated:
+    InvalidPushDeviceTokenError:
+      allOf:
+        - $ref: "#/components/schemas/CodedError"
+        - example:
+            {
+              "code": "INVALID_PUSH_DEVICE_TOKEN",
+              "msg": "Device not recognized",
+              "result": "error",
+            }
+          description: |
+            ## Invalid push device token
+
+            A typical failed JSON response for when the push device token is invalid:
```

Comment:
s/invalid/not recognized by the Zulip server/

Or "this Zulip server".

"Invalid" sounds like it'd be about the token being, like, malformed as an FCM or APNs device token, but that's not what this error is about.